### PR TITLE
BukkitAdapter: Add Bukkit CommandSender --> WorldEdit Actor

### PR DESCRIPTION
**Pull request by JOO200**
I want to use patterns in my plugin for the console.
There is no nice way to adapt a Bukkit CommandSender to a WorldEdit Actor.

### CODE REVIEW

**Comment by octylFractal** [comment]
Technically speaking, this is not necessary because the method is already public. You're free to call it on the plugin yourself.

**Comment by JOO200** [comment]
Yes and no: The method is public but WorldEdit.getInstance() is package private.

**Comment by wizjany** [comment]
notably, WG always has its wrap method on the plugin instead of some adapter class, and it's in the "officially documented" api, so I would probably treat WE the same way.
and yea, you're supposed to get the plugin via plugin manager, as "plugin" methods are dependent on plugin availability (vs just platform availability)

**Comment by me4502** [comment]
IMO if we do this, it also makes sense to include one specifically for players

File: worldedit-bukkit/src/main/java/com/sk89q/worldedit/bukkit/BukkitAdapter.java [hunk 1]
```diff
@@ -119,6 +121,16 @@ public static World adapt(org.bukkit.World world) {
...
+    /**
+     * Create a WorldEdit Actor from a Bukkit CommandSender
+     *
+     * @param sender The Bukkit CommandSender
+     * @return The WorldEdit Actor
+     */
+    public static Actor adapt(CommandSender sender) {
+        return WorldEditPlugin.getInstance().wrapCommandSender(sender);
+    }
+
```
Comment (DarkArc) [inline worldedit-bukkit/src/main/java/com/sk89q/worldedit/bukkit/BukkitAdapter.java]: Should probably allow the other way as well `Actor` -> `CommandSender`.

**Pull request merged**